lib/client/keystore.go

```diff
@@ -366,9 +328,9 @@ func (fs *FSLocalKeyStore) updateKeyWithCerts(o CertOption, key *Key) error {
 
 // CertOption is an additional step to run when loading/deleting user certificates.
 type CertOption interface {
-	// relativeCertPath returns a path to the cert (or to a dir holding the certs)
-	// relative to the session key dir. For use with FSLocalKeyStore.
-	relativeCertPath(idx KeyIndex) string
+	// certPath returns a path to the cert (or to a dir holding the certs)
+	// within the given key dir. For use with FSLocalKeyStore.
+	certPath(keyDir string, idx KeyIndex) string // relativeCertPath returns a path to the cert (or to a dir holding the certs)
```

Comment:
Feels like `// relativeCertPath returns a path to the cert (or to a dir holding the certs)` is a leftover.